### PR TITLE
Single config file please

### PR DESCRIPTION
**Pull request by mayhem**
Just here for running tests..

### CODE REVIEW

File: listenbrainz/webserver/__init__.py [hunk 1]
```diff
@@ -49,37 +49,23 @@ def gen_app(config_path=None, debug=None):
...
-        print("loading consul config file %s)" % os.path.join( os.path.dirname(os.path.realpath(__file__)), '..', 'custom_config.py'))
-        app.config.from_pyfile(custom_config)
+        print("loading consul config file %s)" % config_file)
+        app.config.from_pyfile(config_file)
 
     else:
-        print("loading custom config %s" % custom_config)
-        app.config.from_pyfile(custom_config, silent=True)
-
-    if config_path:
```
Comment (paramsingh) [inline listenbrainz/webserver/__init__.py]: Removing this means that config_path passed to gen_app wouldn't work anymore?

Comment (mayhem) [reply listenbrainz/webserver/__init__.py]: That seems right. Should we keep it? I was just wanting to remove all places where we could get this problem...

Comment (paramsingh) [reply listenbrainz/webserver/__init__.py]: I've never come across a usecase for passing a config path yet, so I'd be okay with removing the functionality.

Comment (mayhem) [reply listenbrainz/webserver/__init__.py]: Its in git -- we can always bring it back if we come across the use case.

**Pull request merged**